modules/installation-cis-ibm-cloud.adoc

```diff
@@ -43,12 +56,23 @@ $ ibmcloud cis instance-create <instance_name> standard <1>
 . Connect an existing domain to your CIS instance:
 
 .. Set the context instance for CIS:
+ifndef::ibm-power-vs[]
 +
 [source,terminal]
 ----
 $ ibmcloud cis instance-set <instance_name> <1>
 ----
 <1> The instance cloud resource name.
+endif::ibm-power-vs[]
+ifdef::ibm-power-vs[]
++
+[source,terminal]
+----
+$ ibmcloud cis instance-set <instance_CRN> <1>
+----
+<1> The instance Cloud Resource Name (CRN).
```

Comment:
The instance CRN ([Cloud Resource Name](https://cloud.ibm.com/docs/account?topic=account-crn))